cymetric/execution.py

```diff
@@ -13,6 +13,7 @@
 import numpy as np
 import pandas as pd
 
+
```

Comment:
Is there a reason for a double space here?

Reply:
pep8 I believe